airflow/www_rbac/templates/airflow/dag_code.html

```diff
@@ -46,13 +46,15 @@ <h5>{{ k }}</h5>
     {{ super() }}
     <script>
       // We blur task_ids in demo mode
-    $( document ).ready(function() {
-      if ("{{ demo_mode }}" == "True") {
-          $("pre span.s").css({
-              'text-shadow': '0px 0px 10px red',
-              'color': 'transparent',
-          });
-      }
+$(document).ready(() => {
```

Comment:
Can we please try to look for `eslint` setting to follow the indentation of the parent jinja template tag? I think It should follow the same indent as the parent `<script>` tag.